apps/Telegram/install

```diff
@@ -10,9 +10,26 @@ function error {
 # Install flatpak first 
 "${DIRECTORY}/pkg-install" "flatpak" "$(dirname "$0")" || error "Failed installing Flatpak!"
 
-sudo -E flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+# Add flathub remote
+( sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo || flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo || error "Flatpak failed to add flathub remote!"  ) | grep -v "Note that the directories 
+
+'/var/lib/flatpak/exports/share'
+'$HOME/.local/share/flatpak/exports/share'
+
+are not in the search path set by the XDG_DATA_DIRS environment variable, so
+applications installed by Flatpak may not appear on your desktop until the
```

Comment:
Did you mean for these lines to be comments?

Reply:
They are in the `grep -v`

Reply:
> They are in the `grep -v`

I guess I depend too much on syntax highlighting! (Also the `grep -v` part was invisible unless I scrolled to the left)